src/clj/rems/api/schema.clj

```diff
@@ -98,6 +98,7 @@
    (s/optional-key :decider) s/Str
    (s/optional-key :decision) s/Keyword
    (s/optional-key :commenters) #{s/Str}
+   (s/optional-key :latest-comment-requests) {s/Str s/Uuid}
```

Comment:
Maybe `:latest-request-by-user`. However this is problematic in that you see other users comment requests so it should only be returned to the "handlers".

Reply:
Hm. I decided to reveal it in the API so that the UI could show the relevant request near the comment action, but yes, this leaks information to too many users if we don't want applicants to see that there are any comment requests.